src/mbgl/map/map.cpp

```diff
@@ -236,6 +236,17 @@ void Map::cancelTransitions() {
     impl->onUpdate(Update::Repaint);
 }
 
+void Map::resetCameraToStyleDefaults() {
```

Comment:
Not really - that only works when we set `cameraMutated` to false _before_ loading a new style. In this approach, we're explicitly resetting the camera _after_ loading the new style.